test/e2e/Makefile

```diff
@@ -41,7 +41,13 @@ E2E_CHART ?= ${REPO_ROOT}/charts/vsphere-cpi
 E2E_DATA_DIR := ${REPO_ROOT}/test/e2e/data
 
 # E2E_DATA_CAPV_VER defines the providers from which version of CAPV to use
-E2E_DATA_CAPV_VER ?= release-1.0
+E2E_DATA_CAPV_VER ?= release-1.9
```

Comment:
@sbueringer Should I change it to main branch of CAPV?

Reply:
Let's stay on release-1.9. Just more stable. Also easier to consume as you can just point to a stable release in the CI config. Otherwise you would have to point "url" to a timestamped nightly build.